package.json

```diff
@@ -0,0 +1,113 @@
+{
+  "name": "react-file-input",
+  "version": "0.1.0",
+  "description": "React FileInput",
+  "engines": {
+    "npm": ">=8.9"
+  },
+  "scripts": {
+    "preinstall": "node tools/nodeVersionCheck.js",
+    "start-message": "node tools/startMessage.js",
+    "prestart": "npm run start-message",
+    "start": "node tools/distServer.js",
+    "predev": "npm run start-message & npm run clean-dist",
+    "dev:example": "node tools/srcServer.js",
+    "lint": "esw --cache webpack.config.* src example tools test features --color",
+    "lint:fix": "npm run lint -- --fix",
+    "lint:watch": "npm run lint -- --watch",
+    "clean-dist": "npm run remove-dist && mkdir dist",
+    "remove-dist": "rm -rf ./dist",
+    "prebuild": "npm run clean-dist",
+    "build": "node tools/build.js",
+    "test": "BABEL_DISABLE_CACHE=1 node --trace-warnings ./tools/runTests.js",
+    "test:cucumber": "rm -rf ./screenshots && NODE_PATH=src wdio wdio.conf.js",
+    "test:coverage": "yarn jest --coverage",
+    "test:watch": "npm run test -- --watch",
+    "selenium:install": "selenium-standalone install",
+    "selenium": "selenium-standalone start"
+  },
+  "author": "Brainhub",
+  "license": "MIT",
+  "dependencies": {
+    "express": "^4.15.2"
+  },
+  "devDependencies": {
+    "autoprefixer": "^7.1.6",
+    "babel-cli": "^6.24.1",
+    "babel-core": "^6.25.0",
+    "babel-loader": "^7.1.1",
+    "babel-plugin-react-display-name": "^2.0.0",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
+    "babel-plugin-transform-react-constant-elements": "^6.23.0",
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.10",
+    "babel-polyfill": "^6.23.0",
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-react-hmre": "^1.1.1",
+    "babel-preset-stage-1": "^6.24.1",
+    "babel-register": "^6.24.1",
+    "browser-sync": "^2.18.12",
+    "chalk": "^2.3.0",
+    "connect-history-api-fallback": "^1.3.0",
+    "css-loader": "^0.28.4",
+    "cucumber": "^2.3.1",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
+    "eslint": "^4.11.0",
+    "eslint-config-brainhub": "^1.6.3",
+    "eslint-watch": "^3.1.3",
+    "extract-text-webpack-plugin": "^3.0.2",
+    "file-loader": "^1.1.5",
+    "history": "^4.7.2",
+    "html-webpack-plugin": "^2.29.0",
+    "isparta": "^4.0.0",
```

Comment:
I see that you configured `jest` so that, you can delete `isparta` and `istanbul` because it won't be used for generating code coverage.